hack/cmd_util.sh

```diff
@@ -156,9 +169,10 @@ function os::cmd::internal::expect_exit_code_run_grep() {
 	local test_eval_func=${4:-os::cmd::internal::success_func}
 
 	os::cmd::internal::init_tempdir
+	os::test::junit::declare_test_start
```

Comment:
You're saying `os::cmd::try_until_success 'os::cmd::expect_success'` ? That' not legal.